slotify/authentication/models.py

```diff
@@ -36,13 +36,17 @@ class User(AbstractBaseUser, PermissionsMixin):
     created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
For username in line 32, just to confirm are we having a separate username vs their true name? If username = real name then it should not be unique

Reply:
Ah, I was thinking that this app must have real name for username. Would a better name be full_name?

Reply:
I think calling it name is fine, but if it is unique then people with the same name cannot register

Reply:
Ah, I must have missed the unique constraint, let me change it!